open-metadata-implementation/access-services/data-engine/data-engine-api/src/main/java/org/odpi/openmetadata/accessservices/dataengine/model/Referenceable.java

```diff
@@ -0,0 +1,148 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.dataengine.model;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.Objects;
+
+public class Referenceable implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private String qualifiedName;
+    private Map<String, String> additionalProperties;
+    private Map<String, String> vendorProperties;
+    private String typeName;
+    private Map<String, Object> extendedProperties;
+
```

Comment:
Fields not present in Referenceable: vendorProperties, typeName, extendedProperties